extensions/google-drive/src/index.tsx

```diff
@@ -158,7 +158,29 @@ export default function Command() {
           ))}
         </List.Section>
       ) : (
-        <List.EmptyView title={isFetching ? "Fetching files, please wait..." : "No files found"} />
+        <List.EmptyView
+          title={isFetching ? "Fetching files, please wait..." : "No files found"}
+          actions={
+            <ActionPanel>
+              {!isFetching && (
+                <>
+                  <Action
+                    title="Reindex Files Cache"
+                    icon={Icon.Hammer}
+                    onAction={reindexFiles}
+                    shortcut={{ modifiers: ["cmd", "shift"], key: "r" }}
+                  />
+                  <Action
+                    title="Clear File Previews Cache"
+                    icon={Icon.Trash}
+                    onAction={clearFilePreviewsCache}
+                    shortcut={{ modifiers: ["ctrl", "shift"], key: "x" }}
+                  />
+                </>
+              )}
+            </ActionPanel>
```

Comment:
Would it make sense to extract these two actions into a reusable component since they're duplicated?

Reply:
https://github.com/raycast/extensions/pull/1541/commits/8923b5fa5545c205db628770d623549a62c2434b

Split as an action components.